internal/todo/repository/sqlite_repository.go

```diff
@@ -0,0 +1,66 @@
+package repository
+
+import (
+	"time"
+
+	models "todo-app/internal/models"
+	"todo-app/internal/todo"
+
+	"github.com/jmoiron/sqlx"
+)
+
+type taskRepo struct {
+	db *sqlx.DB
+}
+
+func NewTaskRepository(db *sqlx.DB) todo.Repository {
+	return &taskRepo{db: db}
+}
+
+func (r *taskRepo) GetTasks() ([]models.Task, error) {
+	var tasks []models.Task
+
+	err := r.db.Select(&tasks, getAllTasks)
+	if err != nil {
+		return []models.Task{}, err
+	}
+	return tasks, nil
+}
+
+func (r *taskRepo) CreateTask(date time.Time, title, comment, repeat string) (int64, error) {
+	res, err := r.db.Exec(insertTask, date.Format("20060102"), title, comment, repeat)
+	if err != nil {
+		return 0, err
+	}
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err

```

Comment:
Лучше все ошибки оборачивать
```suggestion
		return 0, fmt.Errorf("error while getting last id: %w", err)
```